x-pack/plugins/rule_registry/server/routes/get_browser_fields_by_feature_id.ts

```diff
@@ -50,7 +51,7 @@ export const getBrowserFieldsByFeatureId = (router: IRouter<RacRequestHandlerCon
           });
         }
 
-        const browserFields = await alertsClient.getBrowserFields({
+        const browserFields: BrowserFields = await alertsClient.getBrowserFields({
```

Comment:
It's redundant yes

Reply:
I don't see removing it making it harder to know what the type is because the name is exactly like the type (which is great) and IDE and typescript are helpful in this case.